pandas/core/common.py

```diff
@@ -2854,6 +2854,16 @@ def pprint_thing_encoded(object, encoding='utf-8', errors='replace', **kwds):
     value = pprint_thing(object)  # get unicode representation of object
     return value.encode(encoding, errors, **kwds)
 
+def _sizeof(self):
+    """
+    Generates the total memory usage for a object that returns either a value or
+    NDFrame of values
+    """
+    mem = self.memory_usage()
```

Comment:
use ``not lib.isscalar``, then just return (don' need an ``if/else``)

``int(mem.sum())`` should be enough